continuous_integration/environment-3.10-jdk11-dev.yaml

```diff
@@ -4,7 +4,7 @@ channels:
 - nodefaults
 dependencies:
 - dask-ml>=2022.1.22
-- dask>=2022.3.0
+- dask>=2022.3.0, <=2022.7.1
```

Comment:
in theory we should be good to relax the upper pin further to 2022.8.0 though the new rapids release pins to 2022.7.1. I don't think relaxing it further lead to env conflicts on the rapids side

Reply:
Agreed - historically, we've stuck to our own Dask pinnings for release as long as it doesn't cause conflict with RAPIDS release, so I'm okay with bumping this to

```suggestion
- dask>=2022.3.0, <=2022.8.0
```